test/utils.go

```diff
@@ -53,6 +53,9 @@ const aerospikeNs string = "aerospike"
 const zoneKey = "topology.kubernetes.io/zone"
 const regionKey = "topology.kubernetes.io/region"
 
+const tlsPort = 4333
+const nontlsPort = 3000
```

Comment:
nit: Better to use service prefix to denote service tls and non-tls Port
Also, use these constants at other places also in the test-suites where hardcoding is done for these values